content/docs/csidriver/installation/helm/powermax.md

```diff
@@ -232,6 +235,11 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | sidecarProxyImage | Image for csm-authorization-sidecar. | No | " " |
 | proxyHost | Hostname of the csm-authorization server. | No | Empty |
 | skipCertificateValidation | A boolean that enables/disables certificate validation of the csm-authorization server. | No | true |
+| **replication** | [Replication](../../../../replication/deployment) is an optional feature to enable replication & disaster recovery capabilities of PowerMax to Kubernetes clusters.| - | - |
+| enabled                  | A boolean that enables/disables replication feature. |  No      |   false   |
+| image | Image for dell-csi-replicator sidecar. | No | " " |
```

Comment:
pls add default image name .

Reply:
Default image will be set to none , when this value is not provided